tensorflow/lite/g3doc/guide/ops_version.md

```diff
@@ -209,7 +209,7 @@ is supported for every node in Delegation code.
 ```
 const int kMinVersion = 1;
 TfLiteNode* node;
-TfLiteRegistration;
+TfLiteRegistration registration = nullptr;
 context->GetNodeAndRegistration(context, node_index, &node, &registration);
```

Comment:
I think the right fix would be to make sure the result is `kTfLiteOk` here, before proceeding?

Reply:
You are right, I've added `TF_LITE_ENSURE_STATUS` to ensure status is `kTfLiteOk`. Please review again, thanks! :-)